app/style/main.css

```diff
@@ -26,4 +26,93 @@
 
 .bz-item-desc {
     font-weight: bold;
-}
\ No newline at end of file
+}
+
```

Comment:
I think removing these custom styles is best. Let bugzilla style the elements for us.